.gitignore

```diff
@@ -85,6 +85,9 @@ target/
 # pyenv
 .python-version
 
+#pycharm
+.idea
```

Comment:
This belongs in your [global `.gitignore`](https://stackoverflow.com/q/7335420), so please undo this change.